plugins/services/src/js/reducers/JSONSingleContainerParser.js

```diff
@@ -3,7 +3,6 @@ import {
   JSONParser as constraints
 } from "./serviceForm/JSONReducers/Constraints";
 import { JSONParser as container } from "./serviceForm/Container";
-import { JSONParser as containers } from "./serviceForm/Containers";
```

Comment:
Sorry, I missed to explain this is the singleContainerParser. `containers` is a attribute only available in multipleContainers meaning this parser was running on every singleContainerParser without any purpose. Meaning we removed at least one function call here. Which also improves the performance.